R/migrate.R

```diff
@@ -246,3 +255,17 @@ migrate_single <- function(path, config) {
     report_archive_version <- v
   }
 }
+
+
+migrate_clean <- function(config, dry_run) {
+  files <- list.files(file.path(config$root, "archive"),
+                      "^orderly_run_([0-9]+\\.){3}rds",
+                      recursive = TRUE, full.names = TRUE)
+  size <- pretty_bytes(sum(file_size(files)))
+  orderly_log("clean",
+              sprintf("%d backup files to delete (%s)", length(files), size))
+  if (!dry_run) {
+    file.remove(files)
+  }
```

Comment:
Should we confirm that the files have been deleted?
Currently we get 
```
orderly::orderly_migrate(clean=TRUE)
[ clean      ]  1216 backup files to delete (25.75 MB)
```
But orderly does not tell me that the files actually have been deleted.

Reply:
yes, good point!